src/gui/windows/main.zig

```diff
@@ -15,12 +15,16 @@ pub var window = GuiWindow {
 
 const padding: f32 = 8;
 
+fn exitGame(p:usize) void {
+	_ = p;
+	std.process.exit(0);
```

Comment:
This closes the entire application which has some unintended side-effects:
All the `defer ...deinit()` calls in main won't be executed. This means that the settings won't be saved and, important for development, the leak check of zig's general purpose allocator won't be executed.
A better solution would be to close the glfw window(it that possible) or otherwise to add a `running` flag to the main loop.